src/supporting-data/citations/adapters/citationsConverter.ts

```diff
@@ -41,15 +41,65 @@ export type Citation = {
   submissionDatabase?: string;
 };
 
+export type Reference = {
+  citation?: Citation;
+  referencePositions?: string[];
+  referenceComments?: {
+    value: string;
+    type: string;
+  }[];
+  source: { name: string; id?: string };
+  pubMedId?: string;
+  sourceCategories?: string[];
+  referenceNumber?: number; // Only for UniProtKB (trembl and swissprot)
+  communityAnnotation?: CommunityAnnotation; // Only for community annotations
+  annotation?: string; // Only for computationally mapped
+};
+
+export interface CommunityAnnotation {
+  proteinOrGene?: string;
+  function?: string;
+  comment?: string;
+  disease?: string;
+}
+
 export type CitationsAPIModel = {
-  statistics: Statistics;
+  statistics: LiteratureStatistics;
   citation: Citation;
+  references?: Reference[];
 };
 
 export type CitationsUIModel = CitationsAPIModel & {
   // any addition/change by the converter
 };
 
+export const getCitationPubMedId = (citation: Citation) =>
+  citation.citationCrossReferences &&
```

Comment:
```suggestion
  citation.citationCrossReferences?.find((xref) => xref.database === 'PubMed')
```

Reply:
Had to drop coverage after updating this, I guess because things that shouldn't have been rendered in the tests were.

Reply:
You have conflicting files with the sprint branch regarding 2 files that are likely to use this converter, maybe resolving this conflict first before adjusting the coverage would be good, as it's likely that you would need additional tweaking of the coverage value anyway afterwards

Reply:
Yes, addressing comments first and will finish with a merge into this branch so will see then

Reply:
in the end it went back up. All should be addressed now.

Reply:
The previous line is not needed anymore

Reply:
Good catch - I don't think I like using the "commit suggested" that much and prefer to implement the change but maybe that's just me 😄

Reply:
Yeah, you don't need to "accept" the suggestion, I'm mainly using it to show the diff, but that only applies to one line. There's often context to it, like in this case, or when using a different import for example